### PR TITLE
Add component to view game lobby

### PR DESCRIPTION
**Pull request by Ramko9999**
- Added `Lobby.tsx`, a component to display the lobby of players and the host.
- Deleted `/playground`.

Before:
image

After:
image

### CODE REVIEW

File: client/src/components/game/Game.css [hunk 1]
```diff
@@ -59,30 +59,20 @@
...
+  display: flex;
+  flex-direction: row;
+  gap: 1.5rem;
+}
+
 .top-btns {
   display: flex;
   flex: row;
   gap: 1rem;
+  min-height: 4rem;
```
Comment (Ramko9999) [inline client/src/components/game/Game.css:72]: If I didn't set `min-height` to `4em`, this is the UI change when a player joins the lobby:

https://imgur.com/4jPKDpC

File: client/src/components/game/lobby/Lobby.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  flex-direction: row;
+  align-items: flex-end;
+  gap: 0.5rem;
+}
+
+.lobby-avatar {
+  width: 2.2rem;
+  border: 1.5px solid;
+  border-radius: 50%;
+  vertical-align: top;
```
Comment (Ramko9999) [inline client/src/components/game/lobby/Lobby.css:12]: Ran into this weird [issue](https://stackoverflow.com/questions/19212352/div-height-based-on-child-image-height-adds-few-extra-pixels-at-the-bottom). I resolved it by adding `vertical-align: top`.

**Comment by rohanshiva** [comment]
LG to me, do you think we may need to shrink the Avatars if there is more than like certain amount?

https://degen-xyz.vercel.app/components/AvatarGroup

Either way let's merge this for now, and this is more of a "good to have" UI thing.

File: client/src/components/game/lobby/Lobby.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+        return (
+          <div
+            className="lobby-player"
+            onClick={() => {
+              window.open(`https://github.com/${username}`);
+            }}
+            key={index}
+          >
+            <abbr title={abbrevationTitle}>
```
Comment (rohanshiva) [inline client/src/components/game/lobby/Lobby.tsx:50]: you are a css legend now, nice use of `abbr`

Comment (Ramko9999) [reply client/src/components/game/lobby/Lobby.tsx:50]: Learned from the best

**Pull request merged**